frontend/app/components/Session_/EventsBlock/EventsBlock.tsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import cn from 'classnames';
+import { Icon } from 'UI';
+import { List, AutoSizer, CellMeasurer } from "react-virtualized";
+import { TYPES } from 'Types/session/event';
+import { setEventFilter, filterOutNote } from 'Duck/sessions';
+import { show as showTargetDefiner } from 'Duck/components/targetDefiner';
+import EventGroupWrapper from './EventGroupWrapper';
+import styles from './eventsBlock.module.css';
+import EventSearch from './EventSearch/EventSearch';
+import { PlayerContext } from 'App/components/Session/playerContext';
+import { observer } from 'mobx-react-lite';
+import { RootStore } from 'App/duck'
+import { List as ImmList } from 'immutable'
+import useCellMeasurerCache from 'Components/shared/DevTools/useCellMeasurerCache'
+
+interface IProps {
+  setEventFilter: (filter: { query: string }) => void
+  filteredEvents: ImmList<Record<string, any>>
```

Comment:
mayby get rid of Imm Lists right away?

Reply:
in favour of array

Reply:
can't while `session` is present as immutable record (we pull events from it); I'll refactor it at some point but it will take time since we use `session` everywhere

Reply:
ok, should be fixed now